slack/util.js

```diff
@@ -0,0 +1,12 @@
+var SlackUtils = {
+  createPermalink: function(team_domain, channel_name) {
+    if (channel_name) {
+      var timestamp = Date.now() * 1000;
+      return 'https://' + team_domain + '.slack.com/archives/' + channel_name + '/s' + timestamp;
```

Comment:
is that available in config? or would i have to parse it out from the webhook url?

slack makes the team_domain easily accessible in the slash command post body, so i'm inclined to use that: https://api.slack.com/slash-commands